readme.md

```diff
@@ -24,13 +24,34 @@ Anchor is a robust yet lightweight series of UI components, utilities, and style
 
 ### Get Started
 
+**NPM**
+
 ```ssh
 npm i -S @retailmenot/anchor styled-components
 ```
 
-```typescript jsx
+**Yarn**
+
+```ssh
+yarn add -D @retailmenot/anchor styled-components
```

Comment:
Why as a dev dependency?

Reply:
Ugh you're right. Should just be save. Goes to your other point about being more verbose, I would've caught that.